### PR TITLE
Fixed documentation link to edit it

### LINKED ISSUE #125: [bug] Fix documentation link to edit it!

### PR DESCRIPTION
**Pull request by Riverfount**
Close #125

### CODE REVIEW

File: docs/docs/requirements.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# requires
+
+- Python 3.6+
+- Jinja
```
Comment (MarcusMann) [inline docs/docs/requirements.md:4]: Would good you set the package version, Example:

Flask>1.0.0

File: docs/mkdocs.yml [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
 theme:
-  name: 'material'
-  palette:
-    primary: 'indigo'
-    accent: 'indigo'
+  name: readthedocs
+  highlightjs: true
+  hljs_languages:
+    - yaml
+    - rust
```
Comment (MarcusMann) [inline docs/mkdocs.yml:10]: Why rust language?

File: docs/mkdocs.yml [hunk 3]
```diff
@@ -24,4 +26,8 @@ plugins:
       minify_html: true
 
 nav:
-  - Introduction: 'index.md'
+  - Introduction: "index.md"
+  - Requirements: "requirements.md"
```
Comment (MarcusMann) [inline docs/mkdocs.yml:30]: Would cool add requirements page in `index.md`, not?

File: docs/docs/requirements.md (hunk 1 above)
Comment (Riverfount) [reply docs/docs/requirements.md:4]: Done!

File: docs/mkdocs.yml (hunk 2 above)
Comment (Riverfount) [reply docs/mkdocs.yml:10]: It is true! Removed it!

File: docs/mkdocs.yml (hunk 3 above)
Comment (Riverfount) [reply docs/mkdocs.yml:30]: Ok, it is cool, too! Done!

**Pull request merged**